tests/test_trt_compile.py

```diff
@@ -46,7 +60,7 @@ def tearDown(self):
         if current_device != self.gpu_device:
             torch.cuda.set_device(self.gpu_device)
 
-    @SkipIfAtLeastPyTorchVersion((2, 5, 0))
+    @unittest.skipUnless(torch_trt_imported, "torch_tensorrt is required")
```

Comment:
Does this change mean even before 2.5, the test will pass as well? Since we have multiple pytorch version test in the ci.

Reply:
The test will still failed in 24.10 container: https://prod.blsm.nvidia.com/dlmed-clara-jenkins/blue/organizations/jenkins/Monai-latest-image/detail/Monai-latest-image/1306/pipeline/227/

Reply:
In this PR, I used arg_inputs instead of inputs argument in call to torch_tensorrt.convert_method_to_trt_engine, to work around  Torch-TRT bug that forced us to introduce that skip.

Reply:
Hi @borisfom, I don’t think this approach will work with Torch versions prior to 2.4. I can run more tests once you’ve resolved the conflict. Thanks!

Reply:
@KumoLiu - sure please do try with older versions - we may then have to add another guard on minimal Torch version if if fails with < 2.4.